File: src/java/1009.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in); 
        String name = in.next();
        double salary = in.nextDouble();
        double sells = in.nextDouble();

        double total = salary + sells * ((double) 15 / 100);
        System.out.printf("TOTAL = R$ %.2f\n", total);
    }
}

File: src/kotlin/1009.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);
    
    val name: String = sc.next();
    val salary: Double = sc.next().toDouble();
    val sells: Double = sc.next().toDouble();

    val total = salary + sells * (15.0 / 100.0);
    println("TOTAL = R$ ${total.format(2)}");
}

fun Double.format(digits: Int) = "%.${digits}f".format(this)